pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/cache/CacheServiceTest.java

```diff
@@ -13,25 +14,14 @@ public class CacheServiceTest {
     public void testGetInstanceDoesNotThrowNoSuchMethodError() {
         try {
             CacheService.getInstance();
-        } catch (NoSuchMethodError e) {
+        } catch (RedisConnectionException e) {
+            log.info("As expected, cannot connect to local instance that does not exist.");
+        } catch (NoSuchFieldError | NoSuchMethodError e) {
             e.printStackTrace();
-            Assert.fail("Transitive dependency error with redisson: " + e.getMessage());
-        } catch (NoSuchFieldError e) {
             Assert.fail("Could not get CacheService instance: " + e.getMessage());
         } catch (Throwable e) {
-            boolean hasConfigFileError = false;
```

Comment:
Now that this same issue was showing up in the full dev environment, it became clear that the way I was trying to isolate and ignore the fact that the test couldn't connect to localhost was actually obscuring the real problem.